app/konnect/getting-started/configure-mesh.md

```diff
@@ -54,7 +53,7 @@ Perform read-only operations on {{site.mesh_product_name}} resources on Kubernet
 You connect `kumactl` to the global control plane in {{site.konnect_short_name}} so that you can run commands against the control plane.
 
 1. From the left navigation menu in {{site.konnect_short_name}}, open {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager) and select the `example-cp` control plane.
-1. Select **Configure kumactl** from the **Control Plane Actions** dropdown menu and follow the steps in the wizard to connect `kumactl` to the control plane.
+1. Select **Configure kumactl** from the **Global Control Plane Actions** dropdown menu and follow the steps in the wizard to connect `kumactl` to the control plane. <!--How do I Expose the Konnect personal access token as a KONNECT_TOKEN environment variable?-->
```

Comment:
@johnharris85 In the UI wizard to set up kumactl, it mentioned that I'd need to expose the Konnect PAT as a KONNECT_TOKEN environment variable and I wasn't sure how to do that. Do you have a link to docs or could you describe how that's done so I can add it in here?

Reply:
I'll add an instruction in the dialog box in Konnect to show this.